docs/en/core-services/Unified-Namespace.md

```diff
@@ -202,26 +224,133 @@ In `alluxio-site.properties` on master nodes:
+it is more desirable to reduce load on the namenode by disabling active sync and simply using path
+configuration instead. Otherwise, if the load on the namenode isn't a concern, use ActiveSync and
+set the sync interval to -1.
+
+##### Non-HDFS UFS
+
+If a small number of paths are updated regularly or semi-regularly through the UFS directly instead
+of Alluxio, we recommend using the path configuration feature to set the sync interval on the
+specific path(s) based on the understanding of the rate of change in the UFS. This allows files to
+be synced for the special cases without detrimentally affecting the performance of operations on
+other parts of the namespace.
+
+#### UFS frequently updated not through Alluxio
+
+##### HDFS
+
+If the updates happen across many parts (or unknown) of the namespace at a frequent rate, use active
+sync in order to update metadata in a timely manner. If the rate of change is too fast, namespace is
+too large, or the user does not want to update the max RPC size from the namenode, then follow the
+suggestion in the section below for Non-HDFS UFS.
+
+##### Non-HDFS UFS
+
+Set the sync interval for each subtree or the entire cluster based on the understanding of the rate
+of change in the UFS.
+
 ### Other Methods for Loading New UFS Files
 
-The UFS sync discussed previously is the recommended method for synchronizing changes from the UFS.
-Here are some other methods for loading files:
-
-* `alluxio.user.file.metadata.load.type`: This client property can be set to either
-`ALWAYS`, `ONCE`, or `NEVER`. It acts similar to `alluxio.user.file.metadata.sync.interval`,
-but with two caveats:
-    1. It only discovers new files and does not reload modified or deleted files.
-    1. It only applies to the `exists`, `list`, and `getStatus` RPCs.
-    
-    `ALWAYS` will always check the UFS for new files, `ONCE` will use the default
-behavior of only scanning each directory once ever, and `NEVER` will prevent Alluxio
-from scanning for new files at all.
-
-* `alluxio fs ls -f /path`: The `-f` option to `ls` acts is equivalent to setting
-`alluxio.user.file.metadata.load.type` to `ALWAYS`. It discovers new files but
-does not detect modified or deleted UFS files.
-The only way to detect modified or deleted UFS files is to pass the
-`-Dalluxio.user.file.metadata.sync.interval=0` option to `ls`.
+We recommend to configure the sync interval exclusively. It is not recommended to consider
+configuring the metadata load type. However, do note the following equivalencies
+
+`alluxio.user.file.metadata.load.type=NEVER` is equivalent to `alluxio.user.metadata.sync.interval=-1`
+`alluxio.user.file.metadata.load.type=ALWAYS` is equivalent to `alluxio.user.metadata.sync.interval=0`
+
+If the metadata sync interval is configured the metadata load type is ignored.
+
+### Active Sync for HDFS
+In version 2.0, we introduced a new feature for maintaining synchronization between Alluxio space
```

Comment:
nit: newline above this line to be consistent?

Reply:
Updated, thanks.